quickwit-index-config/src/default_doc_mapper/default_mapper.rs

```diff
@@ -738,7 +734,7 @@ mod tests {
             ]
         }"#;
 
-        let builder = serde_json::from_str::<DefaultIndexConfigBuilder>(index_config)?;
+        let builder = serde_json::from_str::<DefaultDocMapperBuilder>(index_config)?;
```

Comment:
Ultra nitpick.

I tend to prefer the style:
```
let builder: DefaultDocMapperBuilder = ...
```

It looks a little bit cleaner, but also it gives you the type of the variable.

Reply:
Yes it's better.